org.bluej.BlueJ.json

```diff
@@ -0,0 +1,73 @@
+{
+  "app-id" : "org.bluej.BlueJ",
+  "branch" : "4.2.0",
```

Comment:
for flathub the branch must be 'stable' but that's set automatically so you should just leave it out.